src/app/components/book-list/book-list.component.ts

```diff
@@ -15,23 +15,32 @@ import { FormActions } from '@app/shared/enums';
   styleUrls: ['./book-list.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class BookListComponent implements OnInit {
+export class BookListComponent implements OnInit, OnDestroy {
 
+    _books: BehaviorSubject<IReadBook[]>;
```

Comment:
1) Pls research about data observable services. In short - it should be in separate service.
2) It should be private.

Reply:
I believe it's resolved now when I use NgRx